dashboard/app/views/lti/v1/integrations.html.haml

```diff
@@ -0,0 +1,37 @@
+#lti-integration-portal
+
+  .row
+    .span10.header
+      %h1= t('lti.integration_portal_header')
+  .row
+    .span10
+      %p= t('lti.integration_portal_instructions').html_safe
+      %p= t('lti.integration_portal_instructions_2').html_safe
+    = form_with(url: :lti_v1_integrations, method: :post, class: "lti-integration-form") do |f|
+      .row
+        .span8
+          .padded-container
+            .row
+              .span3.lti-integration-field-label
+                = f.label :client_id, t('lti.integration_client_id').html_safe
+              .span5
+                = f.text_field :client_id, maxlength: 50
+            .row
+              .span3.lti-integration-field-label
+                = f.label :email, t('lti.integration_email').html_safe
+              .span5
+                = f.email_field :email, maxlength: 255
+            .row
+              .span3.lti-integration-field-label
+                = f.label :lms, t('lti.integration_lms_selector')
+              .span5
+                = f.select :lms, @form_data[:lms_platforms]
```

Comment:
This values displayed in this select dropdown are the keys in the [LMS_PLATFORMS](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/policies/lti.rb#L39) constant. For canvas, this is currently `canvas_cloud`. I'm worried this might be confusing, and am considering renaming this to `canvas`, and we can later add a differentiator for non-cloud Canvas, I.e. self-hosted. Thoughts?

Reply:
I could go either way. Another way to do it that might be more user-friendly is to do a mapping of the keys to capitalized strings for the dropdown, e.g. "Canvas" and "Schoology" - there are some dropdown mappings defined in [application_helper.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/helpers/application_helper.rb). I don't love having these randomly in this helper, but it gets around having underscores and lowercased names.

Reply:
Ok, yeah I was thinking about doing a mapping of some kind. I'll take a look at adding on to application_helper.rb.